apollo-core/src/main/java/com/spotify/apollo/core/ServiceImpl.java

```diff
@@ -515,33 +479,7 @@ public Service build() {
     }
   }
 
-  static class ExecutorServiceCloseable implements Closeable {
```

Comment:
I've had to write code similar to this many times in services where I create my own executors. It would be nice if there was an easy was to add application level executors to the Apollo shutdown cycle, instead of writing this code each time.